assets/js/app.js

```diff
@@ -1,5 +1,8 @@
 import css from '../css/app.css';
+import LiveSocket from "phoenix_live_view"
 
+let liveSocket = new LiveSocket("/live")
+liveSocket.connect()
 // Brunch automatically concatenates all files in your
```

Comment:
2, 4, and 5 don't belong here in this PR and were rolled in with a commit that doesn't talk about them

Reply:
@aonomike Thanks for removing this.